src/pages/trigger-list/trigger-list.jsx

```diff
@@ -193,6 +198,31 @@ class TriggerListPage extends React.Component<Props, State> {
             })}`
         );
     };
+
+    setMetricMaintenance = async (triggerId: string, maintenance: Maintenance, metric: string) => {
```

Comment:
Can you merge this code with similar code that works on trigger page?

Reply:
I have check all duplications, found unused code. Fixed it.